src/client/imagesource.cpp

```diff
@@ -566,7 +566,9 @@ static void apply_hue_saturation(video::IImage *dst, v2u32 dst_pos, v2u32 size,
 		for (u32 x = dst_pos.X; x < dst_pos.X + size.X; x++) {
 
 			if (colorize) {
-				f32 lum = dst->getPixel(x, y).getLuminance() / 255.0f;
+				video::SColor col = dst->getPixel(x, y);
+				f32 lum = (0.3f * col.getRed() + 0.59f * col.getGreen()
+					+ 0.11f * col.getBlue()) / 255.0f;
```

Comment:
These magic values are used twice in the code. I don't think there's any benefit in inlining `getLuminance` in particular.

Reply:
It's back.